feature/experimental/isis/otg_tests/base_adjacencies_test/README.md

```diff
@@ -4,164 +4,204 @@
-    *   interfaces/interfaces/levels/level/adjacencies/adjacency/state/restart-status
-    *   interfaces/interfaces/levels/level/adjacencies/adjacency/state/restart-support
-    *   interfaces/interfaces/levels/level/adjacencies/adjacency/state/restart-suppress
-    *   levels/level/system-level-counters/state/auth-fails
-    *   levels/level/system-level-counters/state/auth-type-fails
-    *   levels/level/system-level-counters/state/corrupted-lsps
-    *   levels/level/system-level-counters/state/database-overloads
-    *   levels/level/system-level-counters/state/exceeded-max-seq-nums
-    *   levels/level/system-level-counters/state/id-len-mismatch
-    *   levels/level/system-level-counters/state/lsp-errors
-    *   levels/level/system-level-counters/state/max-area-address-mismatches
-    *   levels/level/system-level-counters/state/own-lsp-purges
-    *   levels/level/system-level-counters/state/seq-num-skips
-    *   levels/level/system-level-counters/state/spf-runs
-
-*   For LSDB - subpaths of
-
-    *   /network-instances/network-instance/protocols/protocol/isis/levels/level/link-state-database/...
-
-## Protocol/RPC Parameter coverage
-
-*   IS-IS:
-    *   LSP messages
-        *   TLV 1 (Area Addresses)
-        *   TLV 10 (Authentication)
-        *   TLV 22 (Extended IS reach)
-        *   TLV 135 (Extended IP Reachability)
-        *   TLV 137 (Dynamic Name)
-        *   TLV 232 (IPv6 Reachability)
+### Test environment setup
+
+*   DUT has an ingress port and 1 egress port.
+
+    ```
+                             |         |
+        [ ATE Port 1 ] ----  |   DUT   | ---- [ ATE Port 2 ]
+                             |         |
+    ```
+
+### RT-2.1.1 Basic fields test
+
+*   Configure DUT:port1 for an IS-IS session with ATE:port1.
+*   Read back the configuration to ensure that all fields are readable and
+    have been set properly (or correctly have their default value).
+*   Check that all relevant counters are readable and are 0 since the
+    adjacency has not yet been established.
+*   Push ATE configuration for the other end of the adjacency, and wait for
+    the adjacency to form.
+*   Check that the various state fields of the adjacency are reported
+    correctly.
+*   Check that error counters are still 0 and that packet counters have all
+    increased.
+
+### RT-2.1.2 Hello padding test
+
+*   Configure IS-IS between DUT:port1 and ATE:port1 for each possible value
+    of hello padding (DISABLED, STRICT, etc.)
+*   Confirm in each case that that adjacency forms and the correct values
+    are reported back by the device.
+
+### RT-2.1.3 Authentication test
+
+*   Configure IS-IS between DUT:port1 and ATE:port1 With authentication
+    disabled, then enabled in TEXT mode, then enabled in MD5 mode.
+*   Confirm in each case that that adjacency forms and the correct values
+    are reported back by the device.
+
+### RT-2.1.4 ISIS circuit-type point-to-point
```

Comment:
This part looks new/added by this PR and not just a formatting change, right?  Can you add a TODO link which also links to a project issue identifying that this section of the README is not yet implemented in code?  This is needed to help a developer understand what code needs to be added/changed and to ensure that there is a task created to track the work.

(Example of proper TODO link https://github.com/openconfig/featureprofiles/blob/971d8dec564b8302e613295fce2cb0d907cf8935/feature/interface/singleton/otg_tests/singleton_test/README.md?plain=1#L52)